<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PieceWiselinearGrayScale</class>
 <widget class="QWidget" name="PieceWiselinearGrayScale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PieceWiselinearGrayScale</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,1,0">
   <item>
    <widget class="QChartView" name="ChartView_1" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>拐点1</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>X1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="X1_SpinBox"/>
          </item>
          <item>
           <widget class="QSlider" name="X1_Slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Y1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Y1_SpinBox"/>
          </item>
          <item>
           <widget class="QSlider" name="Y1_Slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>拐点2</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>X2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="X2_SpinBox"/>
          </item>
          <item>
           <widget class="QSlider" name="X2_Slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Y2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Y2_SpinBox"/>
          </item>
          <item>
           <widget class="QSlider" name="Y2_Slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sureBtn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
